instana/instrumentation/sqlalchemy.py

```diff
@@ -60,7 +72,7 @@ def receive_dbapi_error(**kw):
                     e = kw['exception']
                     scope.span.set_tag('sqlalchemy.err', str(e))
                 else:
-                    scope.span.set_tag('sqlalchemy.err', "No dbapi error specified.")
+                    scope.span.set_tag('sqlalchemy.err', "No handle error specified.")
```

Comment:
Shall we use `error_event` here instead of hardcoded string?

Reply:
👍 done!